### ISSUE TITLE
[Tool bar] Tooltips and Icon Only variants do not have accessible name

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#2998]
### Provide a general summary of the issue here

Tooltips and Icon Only variants (only buttons using tooltips) of Tool Bar use `<sl-button>` that is described with `aria-describedby` which is not proper way to add accessible name to element. 

<img width="532" height="103" alt="Image" src="https://github.com/user-attachments/assets/cb312b17-08b7-4102-a519-ad2786c42671" />

<img width="529" height="98" alt="Image" src="https://github.com/user-attachments/assets/ecaffefc-000c-4315-8bf5-4a73706221a9" />

### 🤔 Expected Behavior?

`aria-describedby` should be changed to `aria-labelledby` without changing the value of attribute.
Then accessibility name will be present and it will have proper value of text inside the tooltip.

### 😯 Current Behavior

`aria-describedby` is used alone as accessible name. 
axe-core tests report an issue with `aria-command-name` rule.
Some screen readers may have problem reading accessible name for that elements.

### 💁 Possible Solution

As in expected behavior

### 🖥️ Steps to Reproduce

1. On Tool bar > Tooltips and Tool bar > Icon Only pages start axe-core (using DevTools or a11y add-on for Storybook)
2. Check if any issues with name "ARIA command name" or "ARIA commands must have an accessible name" are reported.

### What browsers are you seeing the problem on? (only for bugs in code)

Chrome

### What operating system are you using?

macOS

### 👤 Your name

SLDS team

### 🧢 Your product/team

SLDS Team

### 🎨 Your Theme(s)

Sanoma Learning
